schema/test/unit/mutations/create_project_revision_test.sql

```diff
@@ -62,10 +62,10 @@ select cif.commit_project_revision(1);
 
 select results_eq(
   $$
-  select id, project_id, change_status from cif.create_project_revision(1)
+  select id, project_id, change_status, revision_type from cif.create_project_revision(1, 'Amendment')
   $$,
   $$
-  values (2, 1, 'pending'::varchar)
+  values (2, 1, 'pending'::varchar, 'Amendment'::varchar)
   $$
 );
```

Comment:
Test is missing text about what it is testing

Reply:
We're also missing a test about the default value if the revision type is not provided